assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -103,6 +105,17 @@ export default function SettingsView() {
 							<DisplaySetting value={ ga4PropertyID } />
 						</p>
 					</div>
+					<div className="googlesitekit-settings-module__meta-item">
+						<h5 className="googlesitekit-settings-module__meta-item-type">
+							<VisuallyHidden>
+								{ __( 'Google Analytics 4', 'google-site-kit' ) }
+							</VisuallyHidden>
+							{ __( 'Measurement ID', 'google-site-kit' ) }
```

Comment:
Unfortunately this won't actually work as-expected, because these two strings end up being presented to translators as two disparate strings and thus can't be translated together as-intended. In a language, for instance, where this should read more like "Measurement ID for Google Analytics 4", there'd be no way to properly translate this string.

In order to have a `gettext` string with a React component in it, you'll want to use `createInterpolateElement` from `'@wordpress/element'`.

So what needs to happen is something more like:

```suggestion
							{ createInterpolateElement(
								__( '<visuallyhidden>Google Analytics 4</visuallyhidden> Measurement ID', 'google-site-kit' ),
								{
									visuallyhidden: <VisuallyHidden />,
								}
							) }
```

Reply:
Ah - useful to know, thank you. I've added this to the PR.

Reply:
Oh, that's so frustrating. It's happening because you aren't supplying the component type here, but an instance of it without `children`. `createInterpolateElement` then modifies the component and adds the `children` text to it.

Anyway, `children` should never be a required prop like this, I think that's an oversight. The easy fix, but also the correct one, is to mark `children` as not required by `VisuallyHidden`.

I understand the intent with marking it as a required prop, but it's not necessary.

I'll make the change so we can get this merged, thanks! 👍🏻